src/Avatar/Avatar.js

```diff
@@ -69,11 +70,17 @@ class Avatar extends React.PureComponent {
         `Avatar component prop "color" with the value ${colorProp} is deprecated, and will be removed in next major release, please use instead one of these color: [${avatarColorList.toString()}]`,
       );
     }
-    const color = colorProp || stringToColor(text || name); //if color is provided as a prop use it, otherwise, generate a color based on the text
+    const color = colorProp || stringToColor(text || name); // if color is provided as a prop use it, otherwise, generate a color based on the text
     const sizeNumber = getSizeNumber(size);
     const renderOnHover = !showIndicationOnHover || showIndication;
     const renderIndication =
-      indication && renderOnHover && sizeNumber > minIndicationRenderSize;
+      indication &&
+      renderOnHover &&
+      sizeNumber > minIndicationRenderSize &&
+      !customIndication;
+
+    const renderCustomIndication =
+      customIndication && renderOnHover && sizeNumber > minIndicationRenderSize;
```

Comment:
What do you think about:
```suggestion
    const indicationConstraints = renderOnHover && sizeNumber > minIndicationRenderSize;
    const renderIndication = indicationConstraints && !customIndication && indication;
    const renderCustomIndication = indicationConstraints && customIndication;
```